src/main/java/io/strimzi/test/container/StrimziKafkaCluster.java

```diff
@@ -85,37 +97,104 @@ private void prepareKafkaCluster(final Map<String, String> additionalKafkaConfig
             defaultKafkaConfigurationForMultiNode.putAll(additionalKafkaConfiguration);
         }
 
-        // multi-node set up
+        if (this.useSeparatedRoles) {
+            prepareSeparateRolesCluster(defaultKafkaConfigurationForMultiNode, kafkaVersion);
```

Comment:
What about `prepareDedicatedRolesCluster()`? And overall changing "separate(d)" to "dedicated" overall?

Reply:
Yeah, sounds better.